Synview-frontend/src/components/SyncForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { useGetMyCommitsMutation } from "../services/apiSlice.ts";
+import { useAppSelector } from "../hooks.ts";
+
+export default function SyncForm() {
+  const [githubUsername, setGithubUsername] = useState("");
+  const [projectName, setProjectName] = useState("");
+  const [getCommits] = useGetMyCommitsMutation();
+  const project_id = useAppSelector((state) => state.githubModal.project_id);
+  const user_id = useAppSelector((state) => state.githubModal.user_id);
+
```

Comment:
nit: Would not mix snake_case and camelCase. It's ok to use snake_case as a model name (like `project_id`). But when you refer to it with a JS/TS variable name like this - best to make it `projectId` and `userId`